src/main/java/io/jenkins/plugins/pipelinegraphview/treescanner/PipelineNodeTreeScanner.java

```diff
@@ -101,7 +96,7 @@ public void build() {
      * Gets all the nodes that are reachable in the graph.
      */
     private List<FlowNode> getAllNodes() {
-        heads = execution.getCurrentHeads();
+        List<FlowNode> heads = execution.getCurrentHeads();
```

Comment:
if reversions to this file are made, it would be good to keep this change, now that this is a slightly longer lived object cleaning this reference up sooner is better